### ISSUE TITLE
clang-format documentation has this

### ISSUE THREAD

**Issue by mydeveloperday** [llvm/llvm-project#54207]
![image](https://user-images.githubusercontent.com/16046346/156809813-af03c36c-d1a9-4cb5-94d7-3cedbabb2d65.png)

Not sure why it says '15.0.0git'

**Comment by mydeveloperday** [comment]
related ? https://github.com/llvm/llvm-project/commit/e80c52986e1bb3afa6a92c58b1cb897877923a66

**Comment by mkurdej** [comment]
It's not clang-format specific. IMO it marks that it's not a release documentation.

**Comment by tstellar** [comment]
This is just the version string that the project uses for the main branch.  clang --version will report the same thing.

**Comment by mydeveloperday** [comment]
You right I didn't see it before... it looks very odd! would it be better to say

`15.0.0 (prerelease)` or `15.0.0 (trunk)`

or something other than `git` what has `git` got to do with it?

This just looks bizarre to me.

![image](https://user-images.githubusercontent.com/16046346/156821924-e5e60f1a-3989-42a3-8eb0-20369b9fc728.png)

**Comment by mydeveloperday** [comment]
@tstellar don't get me wrong I like the idea of the documentation having the correct version and no one having to go around and change it every release (that bit is great).

I'm not seeing the "git" that for 'clang-format --version' and if we did I think we might even break some people scripts that check the version because I doubt they'd be expecting to see '0git' at the end

$ clang-format --version
clang-format version 15.0.0 (https://github.com/llvm/llvm-project.git c88deef0a7
218dd5c30500e7d3f58fc23283d3e5)

Is anyone wedded to this idea of having the word "git" or could we transform it into 
```
" (underdevelopment)", 
" (in-progress)", 
" (trunk)", 
" (main branch)"
```

at least in the documentation

**Comment by tstellar** [comment]
I have no preference what the version string is, but if we want to change it I think it would require a community wide RFC.

**Comment by mydeveloperday** [comment]
what there an RFC for the addition of the "git" I could tack onto the back of?

**Comment by tstellar** [comment]
> was there an RFC for the addition of the "git" I could tack onto the back of?

I don't think so.  I suspect that most people don't care about the git suffix, but any time you change the version string, you risk breaking people's scripts, CI, etc.  so that's why I think it needs some kind of higher-visibility than just a patch.

**Comment by mydeveloperday** [comment]
I wasn’t thinking about changing the versioning just have |releasenogit| in the rst
